frontend/src/v4/services/viewer/viewer.tsx

```diff
@@ -753,7 +753,11 @@ export class ViewerService {
 	}
 
 	public async isViewerReady() {
-		return await (this.initialisedPromise && this.initialisedPromise.promise);
+		try {
+			return await this.initialisedPromise.promise;
+		} catch {
+			console.error('Error occurred when checking Viewer readiness')
```

Comment:
Is it possible to trace what causes the error and fix it there instead of logging an error message?

Reply:
lets get of this try. initialization problems will be addressed in #4576 4576

Reply:
will be addressed in #4576